tests/foreman/api/test_bookmarks.py

```diff
@@ -20,13 +20,23 @@
 from requests.exceptions import HTTPError
 from robottelo.constants import BOOKMARK_ENTITIES
 from robottelo.datafactory import invalid_values_list, valid_data_list
-from robottelo.decorators import skip_if_bug_open, tier1
+from robottelo.decorators import bz_bug_is_open, skip_if_bug_open, tier1
 from robottelo.test import APITestCase
 
 
 class BookmarkTestCase(APITestCase):
     """Test for common Bookmark operations via API"""
 
+    @classmethod
+    def setUpClass(cls):
+        """Filter entities list if affected by BZ"""
+        super(BookmarkTestCase, cls).setUpClass()
+        if bz_bug_is_open(1456833):
+            BOOKMARK_ENTITIES[:] = [
+                d for d in BOOKMARK_ENTITIES
```

Comment:
@abalakh @svtkachenko we cannot do that here, you are updating a constant, it will be better to update it at job startup
  in tests/foreman/conftest.py